sidecar/internal/csiaddonsnode/csiaddonsnode.go

```diff
@@ -275,3 +287,65 @@ func generateName(nodeID, namespace, ownerKind, ownerName string) (string, error
 
 	return base, nil
 }
+
+func (mgr *Manager) watchCSIAddonsNodes(scheme *runtime.Scheme, client ctrlClient.Client) error {
```

Comment:
call this from the main right after we call the `Deploy()` can you let the Deploy return the csiaddonsNode object name that can be used to call this function